run-dlp/workflow-definitions/dlp-gcs-file.yaml

```diff
@@ -0,0 +1,65 @@
+main:
+    params: [args]
+    steps:
+    - initVariables:
+        assign:
+        - project: 'uri-test'
+        - topic: 'workflows-demo'
+        - timeInterval: '10'
+    - Create DLP Job:
+        call: http.post
+        args:
+            url: https://us-central1-uri-test.cloudfunctions.net/DlpInspectGcsFile
+            auth:
+                type: OIDC
+            body:
+                project: ${project}
+                bucket: ${args.bucket}
+                filename: ${args.name}
+        result: ResultCreate
+    - Get DLP Job Status:
+        call: http.post
+        args:
+            url: https://us-central1-uri-test.cloudfunctions.net/DlpInspectJobStatus
```

Comment:
Same comment as above about the hard-coded function URL.

Reply:
Same s above about the hard-coded function URL.